plugins/outputs/nats/nats_test.go

```diff
@@ -263,3 +268,106 @@ func createStream(t *testing.T, server string, cfg *nats.StreamConfig) {
 	_, err = js.AddStream(cfg)
 	require.NoError(t, err)
 }
+
+func TestWriteWithLayoutIntegration(t *testing.T) {
```

Comment:
Please move this test above the internal `createStream` function.

Reply:
Done